### ISSUE TITLE
[1.2.0] image_management.rst doctest fails

### ISSUE THREAD

**Issue by olebole** [astropy/ccdproc#565]
When trying to build ccdproc from source (to reproduce #564), I get a doctest error:
```
_________________ [doctest] docs/ccdproc/image_management.rst __________________
042 Note that the name of the file is automatically added to the table as a
043 column named ``file``.
044 
045 Selecting files
046 ---------------
047 
048 Selecting the files that match a set of criteria, for example all images in
049 the I band with exposure time less than 60 seconds you could do::
050 
051     >>> matches = (ic1.summary['filter'] == 'I') & (ic1.summary['exposure'] < 60)
UNEXPECTED EXCEPTION: TypeError("ufunc 'bitwise_and' not supported for the input types, and the inputs could not be safely coe
Traceback (most recent call last):

  File "/usr/lib/python2.7/doctest.py", line 1315, in __run
    compileflags, 1) in test.globs

  File "<doctest image_management.rst[6]>", line 1, in <module>

TypeError: ufunc 'bitwise_and' not supported for the input types, and the inputs could not be safely coerced to any supported 

/tmp/ccdproc-test-JDlDAp/docs/ccdproc/image_management.rst:51: UnexpectedException
------------------------------ Captured log call -------------------------------
image_collection.py        185 DEBUG    keywords in setter before pruning: ['imagetyp', 'object', 'filter', 'exposure'].
image_collection.py        193 DEBUG    keywords after pruning ['filter', 'object', 'imagetyp', 'exposure'].
image_collection.py        207 DEBUG    should be building new table...
image_collection.py        185 DEBUG    keywords in setter before pruning: *.
image_collection.py        193 DEBUG    keywords after pruning ['*'].
image_collection.py        207 DEBUG    should be building new table...
```

The test header:

```
platform linux2 -- Python 2.7.14, pytest-3.2.1, py-1.4.34, pluggy-0.4.0 -- /usr/bin/python2.7
cachedir: ..

Running tests with ccdproc version 1.2.0.
Running tests in lib.linux-x86_64-2.7/ccdproc docs.

Date: 2017-10-17T07:54:22

Platform: Linux-4.13.0-1-amd64-x86_64-with-debian-buster-sid

Executable: /usr/bin/python2.7

Full Python Version: 
2.7.14 (default, Sep 17 2017, 18:50:44) 
[GCC 7.2.0]

encodings: sys: ascii, locale: ANSI_X3.4-1968, filesystem: ANSI_X3.4-1968, unicode bits: 20
byteorder: little
float info: dig: 15, mant_dig: 15

Numpy: 1.13.1
Scipy: 0.19.1
Matplotlib: 2.0.0
Pandas: not available
Astropy: 2.0.2
astroscrappy: 1.0.5
reproject: 0.3.1
Using Astropy options: remote_data: none.
```

**Comment by MSeifert04** [comment]
Puh, I think I've fixed that (https://github.com/astropy/ccdproc/issues/461) a while back but there hasn't been a new version since then (because we're not yet fully astropy 2.0 compatible).

**Comment by olebole** [comment]
OK, when I apply #466, this problem disappears. Thanks for the hint.

**Comment by MSeifert04** [comment]
No problem. Did it also fix #564? I'm not sure but it seems like I did change that test as well in #466.
